test/mocha/helpers.js

```diff
@@ -65,3 +67,78 @@ async function insertTestData(mockData) {
     }
   }
 }
+
+// the `keystores` endpoint uses session based authentication which is
+// mocked
+exports.createKeystore = async ({capabilityAgent, referenceId}) => {
```

Comment:
are you 100% sure that capabilityAgent is the correct name for the key used to create a key store? I'm just checking as I would told there would be different keys one for capabilities and another for key stores.